atom/browser/native_window_views.cc

```diff
@@ -373,6 +373,14 @@ bool NativeWindowViews::IsFullscreen() const {
   return window_->IsFullscreen();
 }
 
+void NativeWindowViews::SetBounds(const gfx::Rect& bounds) {
+  window_->SetBoundsConstrained(bounds);
+}
+
+gfx::Rect NativeWindowViews::GetBounds() {
+  return window_->GetRestoredBounds();
```

Comment:
Windows and other platforms are treated differently, you can reference how  `GetSize` was implemented.